x/gov/stakingtally/stakingtally.go

```diff
@@ -0,0 +1,129 @@
+package stakingtally
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
+	"github.com/cosmos/cosmos-sdk/x/gov/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
```

Comment:
stakingtally shouldn't be in staking module?

Reply:
Yeah, it probably should be actually

Reply:
Do we also need to provide additional param to tally router? In case module could customize tally based on that param.

Reply:
Gov would have no dependency on staking at all then I think

Reply:
It would be nice to reduce dependencies, currently some modules are too related each other.